src/Menu/Menu.tsx

```diff
@@ -22,6 +22,7 @@ export const Menu = React.forwardRef<HTMLDivElement, MenuProps>(
       position = 'vertical-stretched',
       scrollable,
       setPosition = menuDefaultPositionSetter,
+      withOverlay,
```

Comment:
```suggestion
```

Reply:
I need to remove that overlay if `openOnHover` is true.